src/core/ChatSDKError.ts

```diff
@@ -59,15 +61,19 @@ export enum ChatSDKErrorName {
      /** Send message failure */
     ChatSDKSendMessageFailed = "ChatSDKSendMessageFailed",
 
+    AMSClientNotLoaded = "AMSClientNotLoaded",
```

Comment:
These are standard errors that would be exposed to the public. Typically users would leverage them in the code to handle specific scenarios. Would it make sense to use “AttachmentClientNotLoaded” to not expose the internal component?

Reply:
done

Reply:
done